addons/wowl/static/src/action_manager/action_manager.ts

```diff
@@ -821,7 +821,11 @@ function makeActionManager(env: OdooEnv): ActionManager {
   ): Promise<void> {
     const { active_id = null, active_ids = null, active_model = null } =
       options.additionalContext || {};
-    const action = await _loadAction(actionRequest, { active_id, active_ids, active_model });
+    const additionalContext = Object.assign(
+      { active_id, active_ids, active_model },
+      options.additionalContext
+    );
+    const action = await _loadAction(actionRequest, additionalContext);
```

Comment:
I'm not sure this is correct

Reply:
I am not sure either, but this can be simplified.
```js
    const additionalContext = Object.assign(
      {
        active_id: null,
        active_ids: null,
        active_model: null,
      },
      options.additionalContext
    );
    const action = await _loadAction(actionRequest, additionalContext);
```